### PR TITLE
Enable circle 2 and build using linux

### PR DESCRIPTION
**Pull request by wolffan**

### CODE REVIEW

File: Tests/IntegrationTests/IntegrationTests.swift [hunk 1]
```diff
@@ -7,29 +7,39 @@
...
-    func testHelloApiCall() {
+    public func testHelloApiCall() {
         self.checkHost()
     }
     
-    fileprivate func checkHost() {
+    private func checkHost() {
         //given
-        let expectation = XCTestExpectation(description: "Get hello string")
+        let expectation = self.expectation(description: "Get hello string")
```
Comment (wolffan) [inline Tests/IntegrationTests/IntegrationTests.swift]: wow i'm surprised

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: swift:4.0.2
+      - image: mysql:5.7
+        environment:
+          - MYSQL_ROOT_PASSWORD=test
+          - MYSQL_DATABASE=balance
```
Comment (wolffan) [inline .circleci/config.yml]: this creates a database?

**Comment by wolffan** [comment]
All Exchnges tests are failing (in my mac) they pass if I launch tests with swit test. Looking into this

**Comment by wolffan** [comment]
Ok it was my problem :) had to reset the user since it was changed

File: .circleci/config.yml (hunk 2 above)
Comment (einsteinx2) [reply .circleci/config.yml]: Yep it gets created automatically in the mysql docker container.

File: Tests/IntegrationTests/IntegrationTests.swift (hunk 1 above)
Comment (einsteinx2) [reply Tests/IntegrationTests/IntegrationTests.swift]: Open source Swift thing. You have to create/use XCTextExpectations gotten from the XCTest object.

**Pull request merged**